scene/3d/voxel_light_baker.h

```diff
@@ -64,6 +64,29 @@ class VoxelLightBaker {
 		int next_leaf;
 	};
 
+	class xorshift32 {
```

Comment:
A class is probably too much, why not just change the function to receive a pointer to the seed and keep the seed in the inner loop?

Reply:
I'd still have to initialize a seed for each thread. I can only run constructors uniquely using only OpenMP 2.0 constructs. I can make a struct that only holds the RNG state with a small constructor that generates a unique starting point and pass that to the actual RNG function if you prefer though?

Reply:
The private(rng) OMP construct calls xorshift32::xorshift32() once for each thread (in case that wasn't clear)